### PR TITLE
Fix/generate setup commands

### PR DESCRIPTION
**Pull request by k-v-dimitrov**

### CODE REVIEW

File: server/tests/session-helpers.ts [hunk 1]
```diff
@@ -32,7 +34,7 @@ export function generateSetupCommands({
       data: {
         name: `Test profile ${i}`,
         email: `test-${i}@ahaplay.com`,
-        workspace_id: [workshopIndex, "id"],
+        workspace_id: [workspaceIndex, "id"],
         access: i === 0 ? "OWNER" : "TEAM_MEMBER",
         status: "ACTIVE",
         is_completed: true,
```
Comment (k-v-dimitrov) [inline server/tests/session-helpers.ts:40]: Test

Comment (k-v-dimitrov) [inline server/tests/session-helpers.ts:40]: `workkshopIndex` was used where actually workspace index should have been provided. Resolved by adding a separate `workspaceIndex` param.

**Review by FIZLIN — APPROVED** [review]
LGTM, but there are some merge conflicts

**Review by FIZLIN — APPROVED** [review]
LGTM

**Pull request merged**